diff --git a/common-project-config.pri b/common-project-config.pri
index abe6cf6..0607d9d 100644
--- a/common-project-config.pri
+++ b/common-project-config.pri
@@ -57,7 +57,7 @@ exists( meego-release ) {
 
 linux-g++-64|linux-icc-64: INSTALL_LIBDIR = $${INSTALL_PREFIX}/lib64
 else: linux-g++-32|linux-icc-32: INSTALL_LIBDIR = $${INSTALL_PREFIX}/lib32
-else: INSTALL_LIBDIR = $${INSTALL_PREFIX}/lib
+else: INSTALL_LIBDIR = $${INSTALL_PREFIX}/lib64
 
 # default library directory can be overriden by defining LIBDIR when
 # running qmake
